Improve routing documentation (#5450)
containous/traefik: docs/content/providers/file.md

            passHostHeader: false
    ```

## Provider Configuration

> If you're in a hurry, maybe you'd rather go through the [static configuration](../reference/static-configuration/overview.md) and the [dynamic configuration](../reference/dynamic-configuration/file.md) references.

Reviewer: ```suggestion
If you're in a hurry, maybe you'd rather go through the [dynamic configuration for the File provider](../reference/dynamic-configuration/file.md) references.
```
